test/datasets_utils.py

```diff
@@ -170,23 +170,6 @@ def wrapper(self):
     return wrapper
 
 
-def combinations_grid(**kwargs):
```

Comment:
I've moved this to `common_utils` since it is no longer just used by the dataset tests.